makefiles/Makefile.devhub-content

```diff
@@ -56,6 +56,8 @@ next-gen-stage: ## Host online for review
 	mut-publish public ${BUCKET} --prefix=/developer --stage ${ARGS};
 	echo "${COMMIT_HASH}/${MUT_PREFIX}"
 	echo "Hosted at ${URL}/developer";
+	mut-publish public developer-test --prefix="${COMMIT_HASH/MUT_PREFIX}" --stage ${ARGS}
```

Comment:
Should this be the same as line 56?
```suggestion
	mut-publish public developer-test --prefix=/developer --stage ${ARGS}
```

Reply:
we can do it that way to ensure that it goes to the right spot, sure 👍